dev/NumberBox/NumberBox_v1.xaml

```diff
@@ -15,6 +15,9 @@
         <Setter Property="SelectionHighlightColor" Value="{ThemeResource TextControlSelectionHighlightColor}" />
         <contract7Present:Setter Property="SelectionFlyout" Value="{StaticResource TextControlCommandBarSelectionFlyout}" />
         <contract7Present:Setter Property="CornerRadius" Value="{ThemeResource ControlCornerRadius}" />
+        <Setter Property="BorderThickness" Value="{ThemeResource TextControlBorderThemeThickness}"/>
```

Comment:
please make sure you validate the old styles since the test app is using the new styles now.

Reply:
Good call, should be fixed now.